src/cc/frontends/clang/b_frontend_action.cc

```diff
@@ -269,7 +269,37 @@ bool ProbeVisitor::VisitVarDecl(VarDecl *D) {
   }
   return true;
 }
+
+bool ProbeVisitor::IsAddrOfExpr(Expr *E) {
```

Comment:
I had the impression we were going for the `isAddrOfExpr` case (lower case for first letter) for private methods. Is it a coincidence that all private, not-inherited methods are lower case? (I did use an upper case first letter for `IsContextMemberExpr` by mistake.)

Reply:
I actually followed `IsContextMemberExpr`. Looks all other private method indeed starting with lower case. After simplification, I guess I do not need extra function `isAddrOfExpr` since its body becomes so small.